models/issue_comment.go

```diff
@@ -418,6 +420,23 @@ func (c *Comment) LoadAssigneeUser() error {
 	return nil
 }
 
+// LoadResolveDoer if comment.Type is CommentTypeCode and ResolveDoerID not zero, then load resolveDoer
+func (c *Comment) LoadResolveDoer() (err error) {
+	if c.ResolveDoerID == 0 || c.Type != CommentTypeCode {
+		return nil
+	}
+	c.ResolveDoer, err = getUserByID(x, c.ResolveDoerID)
+	if err != nil {
+		return
```

Comment:
The user can be deleted from the system, but the comment must remain resolved.

```suggestion
		if IsErrUserNotExist(err) {
			c.ResolveDoerID = -1
			c.ResolveDoer = NewGhostUser()
			err = nil
		} else {
			log.Error("getUserByID[%d]: %v", c.ID, err)
		}
		return
```